### PR TITLE
Vendor updates

### PR DESCRIPTION
**Pull request by vrothberg**
* pin branches to releases or commits
* remove unused dependencies
* add vendor checks to the CI to make sure everything's in sync

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,10 @@ install:
   # Remove the (brew update) at that time.
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew install gpgme ; fi
 
+before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ; fi
```
Comment (mtrmac) [inline .travis.yml]: What does this do?

Comment (vrothberg) [reply .travis.yml]: Just a silly artifact when playing around :^)

**Comment by vrothberg** [comment]
Got it now. Had to specify the language in the Travis file to have a proper GOPATH. Otherwise, vndr will just delete everything. I also wanted to run it before building the container.

**Comment by vrothberg** [comment]
@mtrmac, I think it's ready to go.

**Comment by mtrmac** [comment]
LGTM… although I didn’t review the `vendor` version changes in detail (the very first one seems to backward quite a bit?!); I do hope someone has reviewed that in libpod.

Thanks!

**Pull request merged**